spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -144,9 +144,16 @@
       end
     end
 
-    context 'without the optional nonce' do
-      let(:nonce) { nil }
-      it { expect(valid?).to eq(true) }
+    context 'nonce' do
+      context 'without a nonce' do
+        let(:nonce) { nil }
+        it { expect(valid?).to eq(false) }
+      end
+
+      context 'with a nonce that is shorter than 32 characters' do
+        let(:nonce) { '123456' }
```

Comment:
I don't feel terribly strongly, but I do think that it's good to reference the constant in the spec since that is the reason behind why this is invalid. Perhaps this would be clearer? 

```
let(:string_less_than_minimum_length) { '1' * (RANDOM_VALUE_MINIMUM_LENGTH -1}
let(:nonce) { string_less_than_minimum_length }
```